core/src/main/java/org/jdbi/v3/core/mapper/reflect/ConstructorMapper.java

```diff
@@ -185,13 +184,13 @@ public T map(ResultSet rs, StatementContext ctx) throws SQLException {
 
         RowMapper<T> mapper = specialize0(ctx, columnNames, columnNameMatchers, unmatchedColumns)
             .orElseThrow(() -> new IllegalArgumentException(String.format(
-                UNMATCHED_CONSTRUCTOR_PARAMETERS, constructor)));
+                UNMATCHED_CONSTRUCTOR_PARAMETERS, factory)));
```

Comment:
These error message constants should be updated to say "constructor or static factory method" instead of just "constructor."